.github/workflows/cgc.yml

```diff
@@ -0,0 +1,19 @@
+name: "Deploy to CGC"
+on:
+  push:
+    branches:
+    - master
```

Comment:
I think this should be on release.

Reply:
So, the latest tag is pulled every time we run a job in CGC, so I dont think release is the right place (as I dont think we want to tag a release to update the deployment file), but youre correct that it shouldnt be every push to master.  I think its best to watch the specific file on master, and only trigger if the `cgc/` directory has changes.  What do you think of that approach?

Reply:
I'm generally not a fan of pulling `latest` for deployed applications. I think it should pull a specific version so that the CGC tool version will produce consistent results.

Reply:
So, yes, i 100% agree with this normally.  I didn't think about using the trigger to automatically bump the version of the docker image in CGC.  This is a really good idea and I have a pretty good idea of how to implement it.

That still doesnt fix the trigger choice discussion though.  Since we dont want to have to trigger a release for every time we want to update the CGC deployment.  We could use a prefixed tag?  Something like `cgc-v0.0.1` and have the action watch for `cgc-v*.*.*`?????

Reply:
I would think we would just do an action that watches for a release. When the release happens, it pushes to CGC. It would be similar to what I've done for DNAnexus in: https://github.com/stjude/CICERO/blob/master/.github/workflows/publish_dx.yml. The Docker build is already pushing tagged versions of the image to the CGC repository.